doc/ref/grpfp.xml

```diff
@@ -107,6 +107,21 @@ A consequence of our convention is that elements of finitely presented
 groups are not printed in a unique way.
 See also <Ref Func="SetReducedMultiplication"/>.
 
+For many <Q>higher level</Q> computations for a finite group,
+such as conjugacy classes or character table,
+it is advisable not to use an FpGroup but an isomorphic group in a better
+internal representation,
+such as a pc group (see <Ref Attr="IsomorphismPcGroup"/>)
+or a permutation group (see <Ref Attr="IsomorphismPermGroup"/>).
+
+In fact,
+calling a function such as <Ref Attr="ConjugacyClasses" Label="attribute"/>
+with an FpGroup that does not store that it is finite
```

Comment:
Maybe “does not already know” would be slightly better than “does not store” since the later might indicate that some fp groups might compute that they are finite but just not store that info.

Reply:
Also shouldn’t FpGroup be “finitely presented group” everywhere here?

Reply:
I think `FpGroup` is better here as it stands for a specific kind of group implementation. PcGroups are also finitely presented but only allow very specific presentations and use special algorithms..

Reply:
Concerning "FpGroup":
The manual chapter starts with the sentence: "A  finitely  presented  group  (in short: FpGroup) is [...]".
And then FpGroup is used in many places. Following Max's argument, one could even replace more occurrences of "finitely presented groups" by "FpGroup".

Concerning "know" vs. "store":
My interpretation is that "store" expresses the technical fact that the flag is set, whereas "know" is more vague in the sense that inside an algorithm, at some point the value of the flag can be deduced from the given information, but it is not set in the object.

Reply:
Good point about "FpGroup", the intention I suppose is that `FpGroup` is written whenever you mean "group belonging to the category `IsFpGroup`". Which sounds fine to me. 

Concerning "know" vs. "store": I agree that "know" is vague, but I think "store" is worse, because it implies that there are finitely presented groups `G` where `IsFinite(G)` can been computed but not stored. If the intention is to say "FpGroups where HasIsFinite returns false" then why not say this explicitly?